ArjunaJTS/trailmap/README.md

```diff
@@ -1,89 +1,76 @@
-JBoss, Home of Professional Open Source
```

Comment:
How come the copyright was removed?

Reply:
the copyright should not be in README. It makes hard to read any information for newcomer. The copyright is part of all the java files so I expect it's not an issue. If it's issue we can add a LICENSE file or something similar. But README is for understanding what the example is good for. Making there mess with copyright is wrong. btw. if you check quickstarts from any other project (e.g. https://github.com/wildfly/quickstart) there is no copyright involved.

Reply:
I think we should leave the copyright in place, but we could put it in a comment as per:
https://github.com/tomjenkinson/quickstart/blob/f304089f4307f747c263c393d6d585c4299f5e61/ArjunaJTS/trailmap/README.md
https://raw.githubusercontent.com/tomjenkinson/quickstart/f304089f4307f747c263c393d6d585c4299f5e61/ArjunaJTS/trailmap/README.md

Reply:
@tomjenkinson  ok, as you wish